symfony/plugins/orangehrmMaintenancePlugin/FormatValueStrategy/FormatWithActivity.php

```diff
@@ -17,33 +17,28 @@
  * Boston, MA 02110-1301, USA
  */
 
+namespace OrangeHRM\Maintenance\FormatValueStrategy;
+
+use OrangeHRM\Core\Traits\ServiceContainerTrait;
+use OrangeHRM\Framework\Services;
+
 /**
  * Class FormatWithActivity
  */
 class FormatWithActivity implements ValueFormatter
 {
-    protected $timesheetService = null;
+    use ServiceContainerTrait;
 
     /**
      * @param $entityValue
-     * @return mixed|string
-     */
-    public function getFormattedValue($entityValue)
-    {
-        return $this->getTimesheetService()->getActivityByActivityId($entityValue)->getName();
-    }
-
-    /**
-     * @return TimesheetService
+     * @return null|string
      */
-    public function getTimesheetService()
+    public function getFormattedValue($entityValue): ?string
     {
-
-        if (is_null($this->timesheetService)) {
-
-            $this->timesheetService = new TimesheetService();
+        $result=$this->getContainer()->get(Services::TIMESHEET_SERVICE)->getActivityByActivityId($entityValue);
```

Comment:
some places not formatted properly. @devishke-orange can you pair and check
```suggestion
        $result = $this->getContainer()->get(Services::TIMESHEET_SERVICE)->getActivityByActivityId($entityValue);
```